kolibri/plugins/coach/assets/src/views/lessons/LessonResourceSelectionPage/content-card/index.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+
+  <router-link :to="link" class="card">
+
+    <div class="content-card">
+      <card-thumbnail
+        class="thumbnail"
+        :thumbnail="thumbnail"
+        :kind="kind"
+        :isMobile="true"
+      />
+
+      <div class="text">
+        <h3 class="title" dir="auto">{{ title }}</h3>
+        <p class="description">
+          <!-- eslint-disable -->
+          <span :class="truncatedClass">{{ descriptionHead }}<span class="visuallyhidden">{{ descriptionTail }}</span></span>
```

Comment:
:flags: some funky stuff happening here for the sake of accessibility. @radinamatic 

Been thinking it might be best to add some more info to the link itself via [the router link](https://github.com/learningequality/kolibri/pull/3195/files#diff-a009e9ea62bdedad35d07112aa563100R3)?